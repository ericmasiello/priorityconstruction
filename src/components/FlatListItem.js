import styled from 'styled-components';
import ListItem from './ListItem';

const FlatListItem = styled(ListItem)`
  &:not(:last-child) {
    margin-right: 0.5rem;
  }
`;

export default FlatListItem;
