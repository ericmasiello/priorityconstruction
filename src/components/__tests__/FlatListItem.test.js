import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FlatListItem from '../FlatListItem';

it('should render <FlatListItem>', () => {
  const component = renderer.create(<FlatListItem />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
