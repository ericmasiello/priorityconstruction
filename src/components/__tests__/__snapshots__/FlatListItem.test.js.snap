// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <FlatListItem> 1`] = `
.c1:not(:last-child) {
  margin-bottom: 0.5rem;
}

.c0:not(:last-child) {
  margin-right: 0.5rem;
}

<li
  className="c0 c1"
/>
`;
